openslides/utils/rest_api.py

```diff
@@ -30,6 +32,69 @@
 router = DefaultRouter()
 
 
+class IdManyRelatedField(ManyRelatedField):
+    field_name_suffix = '_id'
+
+    def bind(self, field_name, parent):
+        self.source = field_name[:-len(self.field_name_suffix)]
+        super().bind(field_name, parent)
+
+
+class IdPrimaryKeyRelatedField(PrimaryKeyRelatedField):
+    """
+    Field, that renames the field name to FIELD_NAME_id.
+
+    Only works together the our ModelSerializer.
+    """
+    field_name_suffix = '_id'
+
+    def bind(self, field_name, parent):
+        """
+        Called when the field is bound to the serializer.
+
+        Changes the source  so that the original field name is used (removes
+        the _id suffix).
+        """
+        if field_name:
+            self.source = field_name[:-len(self.field_name_suffix)]
+        super().bind(field_name, parent)
+
+    @classmethod
+    def many_init(cls, *args, **kwargs):
+        """
+        Method from rest_framework.relations.RelatedField That uses our
+        IdManyRelatedField class.
+        """
+        # TODO: this method can be removed if this is merged and released:
+        # https://github.com/tomchristie/django-rest-framework/pull/3122
+        list_kwargs = {'child_relation': cls(*args, **kwargs)}
+        for key in kwargs.keys():
+            if key in MANY_RELATION_KWARGS:
+                list_kwargs[key] = kwargs[key]
+        return IdManyRelatedField(**list_kwargs)
+
+
+class ModelSerializer(_ModelSerializer):
+    """
+    ModelSerializer that changes the field names of related fields to
+    FIELD_NAME_id.
+    """
+    serializer_related_field = IdPrimaryKeyRelatedField
+
+    def get_fields(self):
+        fields = super().get_fields()
```

Comment:
Can you please add a docstring and maybe rename new_fields to manipulated_fields or customized_fields or something like that so we don't think these fields are *newley created/initiated* but only some of them are changed/renamed in a special way?